HomeTask3/HomeTask3/translationIntoNewSystem.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Text;
+
+namespace DevTask3
+{
+    /// <summary> 
+    /// This class creates a string that is a translation of a number into another number system
+    /// </summary>
+    public class TranslationIntoNewSystem
+    {
+        private long numberForConversion;
+        private long foundationOfNewSystem;
+
+        public TranslationIntoNewSystem(string[] valuesNumberAndBaseOfSystem)
+        {
+            numberForConversion = Convert.ToInt64(valuesNumberAndBaseOfSystem[0]);
+            foundationOfNewSystem = Convert.ToInt64(valuesNumberAndBaseOfSystem[1]);
+        }
+
+
+        /// <summary> 
+        /// Method CreateSubstringOfEvenElements
+        /// A method that creates a string that is a representation of a number in the new number system
+        /// </summary> 
+        /// <returns>Value In New System</returns>
+        public string CreatingRepresentationOfNumberInNewSystem()
+        {
+            StringBuilder valueInNewSystem = new StringBuilder();
+            long nextNumberToAdd;
+            do
+            {
+                nextNumberToAdd = numberForConversion % foundationOfNewSystem;
+                valueInNewSystem.Append(ConvertingFigureToRepresentationInAnotherSystem(nextNumberToAdd));
+                numberForConversion = numberForConversion / foundationOfNewSystem;
+            } while (0 != numberForConversion);
```

Comment:
check in codestyle whether while should be on new line or not

Reply:
![image](https://user-images.githubusercontent.com/36936319/37090187-65ede5d8-2214-11e8-8cd0-5f87d3bdfb14.png)
should not be on a new line